src/lib/ngsi/AttributeList.cpp

```diff
@@ -110,6 +112,25 @@ void AttributeList::release(void)
 
 /* ****************************************************************************
 *
+* lookup - 
+*/
+bool AttributeList::lookup(const std::string& attributeName)
```

Comment:
Yeah, I'm sure we have lots of 'forgotten' functions out there that could be implemented as a method for the ngsi-request classes instead. Making these functions methods it is easier to find and use them later